docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,272 @@
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide documents any necessary extra preparation for airgap deployments,
+as well the steps that are needed to successfully deploy Canonical Kubernetes
+in such environments.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta --basename k8s
+sudo snap download core20 --basename core20
+```
+
+The [core20][Core20] and `k8s` snap are downloaded. `core20.assert` and 
+`k8s.assert` are necessary to verify the integrity of the snap packages.
+
+```{note} 
+Update the version of k8s by adjusting the channel parameter.
+For more information on channels visit the
+[channels explanation](/snap/explanation/channels.md).
+```
+
+```{note}
+With updates to the snap the base core is subject to change in the future.
+```
+
+### Prep 2: Network Requirements
+
+Air-gap deployments are typically associated with a number of constraints and
+restrictions when it comes to the networking connectivity of the machines.
+Below we discuss the requirements that the deployment needs to fulfill.
+
+#### Cluster node communication
+<!-- TODO: Add Services and Ports Doc -->
+Ensure that all cluster nodes are reachable from each other.
+<!-- Refer to [Services and ports][svc-ports] used for a list of all network
+ports used by Canonical Kubernetes.  -->
+
+
+#### Ensure proxy access
+
+This section is only relevant if access to upstream image registries
+(e.g. docker.io, quay.io, rocks.canonical.com, etc)
+is only allowed through an HTTP proxy (e.g. squid).
+
+Ensure that all nodes can use the proxy to access the image registry.
+For example, if using `http://squid.internal:3128` to access docker.io,
+an easy way to test connectivity is:
+
+```
+export https_proxy=http://squid.internal:3128
+curl -v https://registry-1.docker.io/v2
+```
+
+Please refer to the next section `images` on how to use the HTTP proxy
+to allow limited access to image registries.
+
+### Prep 3: Images
+
+All workloads in a Kubernetes cluster are running as an OCI image.
+Kubernetes needs to be able to fetch these images and load them
+into the container runtime, in order to run workloads.
+For a Canonical Kubernetes deployment, you will need to fetch the images
+used by its features (network, dns, etc) as well as any images that are
+needed to run your workloads.
+
+The following options are presented in the order of
+increasing complexity of implementation.
+You may also find it helpful to combine these options for your scenario.
+
+If you already have a running cluster,
```

Comment:
You don't need a running cluster to execute this command. It is enough to just snap install it.